packages/create-react-app/index.js

```diff
@@ -105,7 +105,7 @@ function createApp(name, verbose, version) {
   checkAppName(appName);
 
   if (!pathExists.sync(name)) {
-    fs.mkdirSync(root);
```

Comment:
@gaearon For that, we need to add https://www.npmjs.com/package/fs.extra as a dependency. Actually, I found few libraries for the same purpose (maybe this too). But, I thought of not adding since it's an additional dependency. 

What do you think?